### PR TITLE
New company

### PR DESCRIPTION
**Pull request by ricardokdz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ Name | Website
 Abacomm | http://www.abacomm.com.br/
 Ampla Visão | http://www.amplavisao.com/freelascad
 Avante.com | https://avante.com
+Brave Investments | https://github.com/BraveInvestments
```
Comment (tauil) [inline README.md]: Don't you have a company website instead of github page?

**Comment by ricardokdz** [comment]
No, we are developing.

**Comment by lerrua** [comment]
:+1:

**Comment by tauil** [comment]
:+1:

**Pull request merged**